numpy/lib/nanfunctions.py

```diff
@@ -25,12 +25,11 @@
 import numpy as np
 from numpy.lib import function_base
 
-
 __all__ = [
     'nansum', 'nanmax', 'nanmin', 'nanargmax', 'nanargmin', 'nanmean',
     'nanmedian', 'nanpercentile', 'nanvar', 'nanstd', 'nanprod',
     'nancumsum', 'nancumprod'
-    ]
+]
```

Comment:
Total nitpick: can we either not change this (and not remove the empty line above), or move the `]` to the end of the list, i.e., behind `nancumprod`?